coherence-spring-data/src/main/java/com/oracle/coherence/spring/data/repository/query/CoherenceRepositoryQuery.java

```diff
@@ -164,11 +165,46 @@ public Object execute(Object[] parameters) {
 		return result;
 	}
 
+	@SuppressWarnings("unchecked")
+	@Nullable
+	private Object executeProjectingQuery(ReturnedType returnedType, QueryResult q, ResultProcessor processor) {
+		Class<?> resultType = returnedType.getReturnedType();
+		List result = resultType.isInterface()
+				? fetchWithInterfaceProjection(q.getFilter(), resultType, returnedType.getDomainType())
+				: fetchWithClassProjection(q.getFilter(), resultType);
+
+		Sort sort = q.getSort();
+		Comparator comparator = null;
+		if (sort.isSorted()) {
```

Comment:
This block could leverage `com.oracle.coherence.spring.data.support.Utils.toComparator(Sort)`; this was added recently due to some refactoring I had performed.